libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -407,7 +407,7 @@ public void setContent(CharSequence text) {
             return;
         }
 
-        content.fromHtml(text.toString());
+        content.fromHtml(removeVisualEditorProgressTag(text.toString()));
```

Comment:
Is there a way to only call this routine when the content is first set instead of every time it's set? Also, it might be more optimal to first check whether the string contains "<progress" before calling the regex routine.

Reply:
> Also, it might be more optimal to first check whether the string contains "<progress" before calling the regex routine.
addressed this one in bac86b5

Unfortunately, making sure this is the first time we've changed from Visual Editor to Aztec would have to be done on a per-post basis, that means we'd need to keep a table that tracks this behavior. It's also going to be removed once we make Aztec available as the default editor.